models/user.py

```diff
@@ -27,6 +27,33 @@ def get_id(self):
     from models import draft
     pick = relationship('Draft', back_populates="team", uselist=False)
 
+    @classmethod
+    def standings(cls, dbsession):
+        users = dbsession.query(User).all()
+
+        users_w_total_score = {}
+        for user in users:
+            users_w_total_score[user] = user.team_score
+
+        users_sorted_by_score = sorted(users_w_total_score, key=lambda u: users_w_total_score[u])
```

Comment:
if im reading this right, i dont think you need `users_w_total_score`. you can just do
`users_sorted_by_score = sorted(users, key=lambda u: u.team_score)` (or just reassign back to `users`)